src/Pages/Login/Login.js

```diff
@@ -96,6 +132,17 @@ export default function Login() {
           </button>
         </form>
       </div>
+      <ToastContainer
```

Comment:
You don't need to use this toast container component everytime I'm seperate files.
Just use it in app.js and it will be available for all.

Reply:
done